blockstack/blockstack-app
# refactor: onboarding after install

> Try out this version of the Stacks Wallet - download [extension builds](https://github.com/blockstack/ux/actions/runs/966342490).

This PR changes the onboarding flow so that now after install the user is directed back to their app tab and it reloads the page so the extension can be detected as installed.

Issues:
https://github.com/blockstack/connect/issues/144
https://github.com/blockstack/connect/issues/98

Testing:
1. Point at the alpha release changes to connect linked in the PR above.
2. Run the test-app
3. Load the unpacked version into Chrome
4. Going thru onboarding in the full-page wallet
5. After you set your password, the ext should automatically redirect you back to the app tab
6. Click on `Sign up` again
7. Choose an account
8. Done

I think ultimately the goal is to extract out the concept of the `Sign up` button in the test-app and make it available via the connect library (@jasperjansz new designs for onboarding), but this is a good first step.

cc/ @aulneau @kyranjamie @fbwoolf
